src/v/storage/kvstore.h

```diff
@@ -11,7 +11,9 @@
 
 #pragma once
 #include "base/seastarx.h"
+#include "bytes/bytes.h"
 #include "bytes/iobuf.h"
+#include "container/chunked_hash_map.h"
 #include "metrics/metrics.h"
 #include "storage/fwd.h"
 #include "storage/ntp_config.h"
```

Comment:
good to remove the btree include further down?

Reply:
Oops, fixed!